purchase_exception/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Purchase Exception',
+    'summary': 'Custom exceptions on purchase order',
+    'version': '12.0.1.0.0',
+    'category': 'Generic Modules/Purchase',
+    'author': "Akretion, Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com',
```

Comment:
replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)